### PR TITLE
added function log out using server

### PR DESCRIPTION
**Pull request by EvgeniyKoren**
Logout with BE

### CODE REVIEW

File: src/header/HeaderButtons.js [hunk 1]
```diff
@@ -5,12 +5,11 @@ import './header.css';
...
 import { Link } from 'react-router-dom';
 import { Overlay, Popover, Button, Row } from 'react-bootstrap';
-
+import axios from 'axios'; 
 import './loginmenu.css';
 import { useState, useRef } from 'react';
 
 
-
 function Greeting(props) {
```
Comment (fazlmakhsudov) [inline src/header/HeaderButtons.js:13]: Is it possible that file should contain one component and inside there are all internal methods and fields?

File: src/header/HeaderButtons.js [hunk 2]
```diff
@@ -19,8 +18,6 @@ function Greeting(props) {
 }
 
 
-
-
 class NotLogedIn extends React.Component {
```
Comment (fazlmakhsudov) [inline src/header/HeaderButtons.js:21]: same aforementioned note

File: src/header/HeaderButtons.js [hunk 3]
```diff
@@ -121,7 +133,7 @@ function UserLoggedIn(props) {
 								</Button>
```
Comment (fazlmakhsudov) [inline src/header/HeaderButtons.js:133]: Man, it is good style to format code. Just note))

File: src/header/HeaderButtons.js (hunk 1 above)
Comment (EvgeniyKoren) [reply src/header/HeaderButtons.js:13]: @fazlmakhsudov , it woulbe correct, but it's not my component. I prefer don't change code that wasn't written by me if there is no use

**Pull request merged**